css/style.css

```diff
@@ -346,11 +336,14 @@ footer {
   grid-row-start: 2;
   display: grid;
   grid-template-columns: repeat(4, 1fr) 150px;
+  padding: 30px;
+  padding-top: 0;
```

Comment:
An aside from this [comment](https://github.com/electrolyfish/lightbeam-we/pull/112#issuecomment-318718783) on the PR, since `footer`'s container element has padding, reverting this (aka removing these two padding lines) would keep the whitespace consistent as part of the fix.